test/conftest.py

```diff
@@ -311,7 +313,12 @@ def pudl_sql_io_manager(
     """
     logger.info("setting up the pudl_engine fixture")
     if not live_dbs:
-        reset_db()
+        db_path = pudl_settings_fixture["pudl_db"]
+
+        # Create the database and schemas
+        engine = sa.create_engine(f"sqlite:///{db_path}")
+        md = Package.from_resource_ids().to_sql()
+        md.create_all(engine)
```

Comment:
Does it make sense to call the alembic python api to setup the database?

Reply:
I tried it and ran into some surprising behavior, though maybe I'll try again and see if that works better than a direct create_all.